ToolsForMaintainersOfTheProjectTemplate/Scintilla_iface_synchronizer/Face.py

```diff
@@ -50,6 +50,10 @@ def ReadFromFile(self, name):
 		currentCategory = ""
 		currentComment = []
 		currentCommentFinished = 0
+		import os, os.path, inspect
```

Comment:
Changes cwd to the script path so that relative navigation paths work correctly such as ../../ etc.